ui/app/AppLayouts/Profile/views/NotificationsView.qml

```diff
@@ -1,399 +1,576 @@
-                onClicked: {
-                    localAccountSensitiveSettings.notifyOnNewRequests = checked
+            StatusBaseText {
+                Layout.preferredWidth: root.profileContentWidth
+                Layout.leftMargin: Style.current.padding
+                text: qsTr("Notification Content")
+                font.pixelSize: Constants.settingsSection.subHeaderFontSize
+                color: Theme.palette.directColor1
+            }
+
+            NotificationAppearancePreviewPanel {
+                id: notifNameAndMsg
+                Layout.preferredWidth: root.profileContentWidth
+                Layout.leftMargin: Style.current.padding
+                name: qsTr("Show Name and Message")
+                notificationTitle: "Vitalik Buterin"
+                notificationMessage: qsTr("Hi there! So EIP-1559 will defini...")
+                buttonGroup: messageSetting
+                checked: localAccountSensitiveSettings.notificationMessagePreviewSetting === Constants.settingsSection.notificationsBubble.previewNameAndMessage
+                onRadioCheckedChanged: {
+                    if (checked) {
+                        localAccountSensitiveSettings.notificationMessagePreviewSetting = Constants.settingsSection.notificationsBubble.previewNameAndMessage
+                    }
                 }
             }
 
-            // TODO: replace with StatusListItem
-            StatusSettingsLineButton {
-                //% "Muted users"
-                text: qsTrId("muted-users")
-                currentValue: root.notificationsStore.mutedContactsModel.count > 0 ?
-                                  //% "None"
-                                  root.notificationsStore.mutedContactsModel.count : qsTrId("none")
-                isSwitch: false
-                onClicked: {
-                    const mutedChatsModal = notificationsContainer.mutedChatsModalComponent.createObject(notificationsContainer)
-                    mutedChatsModal.model = root.notificationsStore.notificationsModule.mutedContactsModel
-                    //% "Muted contacts"
-                    mutedChatsModal.title = qsTrId("muted-contacts");
-                    //% "Muted contacts will appear here"
-                    mutedChatsModal.noContentText = qsTrId("muted-contacts-will-appear-here");
-
-                    mutedChatsModal.unmuteChat.connect(function(chatId){
-                        root.notificationsStore.unmuteChat(chatId)
-                    })
-
-                    mutedChatsModal.open();
+            NotificationAppearancePreviewPanel {
+                Layout.preferredWidth: root.profileContentWidth
+                Layout.leftMargin: Style.current.padding
+                name: qsTr("Name Only")
+                notificationTitle: "Vitalik Buterin"
+                //% "You have a new message"
+                notificationMessage: qsTrId("you-have-a-new-message")
```

Comment:
We no longer use qsTrId iirc. You can switch it to a qsTr

Reply:
@jrainville didn't know that. Thanks.